packages/core/helper-plugin/src/components/GenericInput.jsx

```diff
@@ -441,13 +451,18 @@ const GenericInput = ({
     }
     default: {
       return (
```

Comment:
What do you think about adding a comment here, this branch renders "not supported"? Previously the name of the component made that clear for me...

Reply:
✅